witness/golang/README.md

```diff
@@ -0,0 +1,32 @@
+Go Witness
+==============
+
+The witness is an HTTP service that stores checkpoints it has seen from
+different verifiable logs in a sqlite database.  This is a very lightweight way
+to help detect or even prevent split-view attacks.
+
+Once up and running, the witness provides three API endpoints (as defined in
+`api/http.go`):
+- `/witness/v0/logs` returns a list of all logs for which the witness is
+  currently storing a checkpoint.
+- `/witness/v0/logs/<logid>/update` acts to update the checkpoint stored for 
+  `logid`.
+- `/witness/v0/logs/<logid>/checkpoint` returns the latest checkpoint for
+  `logid`, signed by the witness.
+
+Running the witness
```

Comment:
I think it's worth mentioning that this cannot easily be run as a replicated job because it uses sqlite, which is a local file. For better scaling/reliability this would require changing to use a server-based DB, but that isn't planned as yet.

Reply:
Cool, good point.  Added a little mention now.